examples/get_schedule_block_info.py

```diff
@@ -79,6 +79,13 @@ def main():
         #              u'outcome': u'UNKNOWN',
         #              u'data_quality': None}
 
+    cb_ids = ['1555494792.25']
+    if len(cb_ids) > 0:
+        cb_detail = yield portal_client.sb_ids_by_capture_block(cb_ids[0])
```

Comment:
All the changes to this example script were just used for testing.  They do not relate to the script's original purpose of "getting schedule block info".  Please remove them.  This PR should not be changing this example.

If you think an example of getting SBs linked to a  CB is necessary, then a new example script dedicated to that should be created.